perftracker/static/js/perftracker.js

```diff
@@ -990,4 +990,81 @@ function pt_isInViewport(elem) {
 
 function pt_pluralize(count, noun, suffix = 's') {
     return `${count} ${noun}${count !== 1 ? suffix : ''}`;
+}
+
+function pt_charts_analyze() {
+    if (hide_charts_information) {
+        alert('Not implemented yet!');
+        return;
+    }
+    if (analyze_was_called) return;
+    analyze_was_called = true;
+
+    var payload = {};
+    Object.keys(sect_data).forEach(s_id => {
+        payload[s_id] = sect_data[s_id].series['0'].data;
```

Comment:
series['0']: 
(1)  0, not '0'
(2) we should probably warn user if there is > 1 jobs in comparison